grails-app/migrations/0.9.x/changelog-2023-11-06-1230-add-missing-role-names.xml

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+  <changeSet author="drodzewicz" id="061120231230-0">
+    <preConditions onFail="MARK_RAN">
+      <sqlCheck expectedResult="1">
+        SELECT COUNT(*) FROM role WHERE role_type = "ROLE_ADMIN" AND (name IS NULL OR name = "");
```

Comment:
Use single quotes if possible. I encountered an issue with DigitalOcean's database service in which it uses ANSI quotes SQL Mode setting that causes issues when using double quotes. It might be common for this setting to be configured with some of the cloud providers, so using single quotes will help us avoid unnecessary headaches.

https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html#sqlmode_ansi_quotes